libexec/bats

```diff
@@ -26,35 +26,58 @@ help() {
   echo
 }
 
+BATS_READLINK=
+
 resolve_link() {
-  $(type -p greadlink readlink | head -1) "$1"
+  if [[ -z "$BATS_READLINK" ]]; then
```

Comment:
Is there a compelling reason to use the `[[` bash extension here? I believe we're attempting to be POSIX compliant with bats, yes? So that bats can be used on minimal POSIX CI containers and whatnot?

Reply:
I'm not aware of any directive to remain strictly POSIX compliant in Bats. If I'm mistaken, I'm happy to look into changing it; but `[[ ]]` is a bit more powerful. (At any rate, `$()` for command substitutions isn't POSIX compliant, as far as I can tell.)